src/init-ceph.in

```diff
@@ -43,7 +38,6 @@ if [ -n "$CEPH_BIN" ] && [ -n "$CEPH_ROOT" ] && [ -n "$CEPH_BUILD_DIR" ]; then
     SBINDIR=$CEPH_ROOT/src
     ETCDIR=$CEPH_BIN
     LIBEXECDIR=$CEPH_ROOT/src
-    SYSTEMD_RUN=""
```

Comment:
@odivlad this script is also used by developer to start/stop daemons in a test cluster, see https://github.com/ceph/ceph/blob/master/README.md#running-a-test-cluster and #9897 . so maybe we can keep this part?